### ISSUE TITLE
ai update command steals the whole input

### ISSUE THREAD

**Issue by hdkiller** [BuilderIO/ai-shell#50]
Hello,

The "update" command steals the user input:

```
ai update all packages

Running: npm update -g @builder.io/ai-shell


up to date in 426ms

18 packages are looking for funding
  run `npm fund` for details
```

Another example:

```
 ai update current time

Running: npm update -g @builder.io/ai-shell

```

Should only run when there is no other arguments passed.

**Comment by uripeled2** [comment]
I think you should do ai 'update current time'

**Comment by steve8708** [comment]
ah great point, let me refactor this to only match exactly `ai update`. and agreed @uripeled2 that's the best workaround right now, and I'll fix this up to be more intuitive and not capture anything that starts with `update` too

**Comment by steve8708** [comment]
should be fixed now in `0.1.14` - thanks for reporting!
